Added dialogue 4 dictionary to examples

Al-Kom opened the pull request.

Al-Kom committed:
Added dialogue dictionary_4 to examples

Al-Kom committed:
Move dialog 4 to separated file

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,601 @@
+    (*
+        -> rrel_gramm_number_singular: rrel_gramm_case_subjective: rrel_gramm_gender_masculine: ...
+        (*
+            -> rrel_standard: [хороший];;
```
Вроде как речь идёт о английском языке, при чём тут русский?

Al-Kom replied:
Так было в примере, меня это тоже смутило

NikiforovSergei replied:
Возможно возникло недопонимание, но все же тут должен быть английский.

PlagaMedicum commented:
Сейчас вообще под вопросом отношение rrel_standard, почему не указывается язык, но вообще, нужно англ добавить. Тут либо всё менять на англ, либо как-то добавлять его к тому, что есть, но не знаю, каким образом это осуществимо

PlagaMedicum reviewed (commented):
Сейчас вообще под вопросом отношение rrel_standard, почему не указывается язык, но вообще, нужно англ добавить. Тут либо всё менять на англ, либо как-то добавлять его к тому, что есть, но не знаю, каким образом это осуществимо

PlagaMedicum reviewed (changes requested):
Исправить системные идентификаторы лексем. Поменять эталоны лексем и основные идентификаторы на англоязычные. Сделать эталоны лексем и основные идентификаторы прописными.

Al-Kom committed:
Update dialog 4 dictionary

Update dictionary of four'th dialog after LAES (Lexical analysis
of English sentences) rules changing

Al-Kom commented:
Словарь обновлен в соответствии с измененными правилами лексического анализа англоязычных предложений

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,654 @@
+    <- lexeme_arbitrary; <- lexeme_qualitative;
+    <- lexeme_multiple; <- lexeme_personal;;
+
+l_en_ve_are
```
Стоп, ты описываешь лексемы или словоформы?

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,654 @@
+    <- lexeme_arbitrary; <- lexeme_qualitative;
+    <- lexeme_single; <- lexeme_personal;;
+
+l_en_ve_am
```
то же самое, это же словоформа

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,654 @@
+    <- lexeme_nominative; <-lexeme_qualitative;
+    <- lexeme_single;;
+
+l_en_ve_search_for
```
?

Al-Kom replied:
search -- поиск
search for -- искать

PlagaMedicum replied:
сейчас мы не лезем в словосочетания итп. Задача сейчас разобраться со словоформами и лексемами. Плюс, то, что ты написал в комментарии — часть реального состояния дел

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,654 @@
+    <- lexeme_nominative; <-lexeme_qualitative;
+    <- lexeme_multiple;;
+
+l_en_pn_this_one
```
??

Al-Kom replied:
Здесь one используется не в значении "один". Вместе с this они означают "этот самый", "вот этот"

PlagaMedicum replied:
про словосочетания написал выше

PlagaMedicum reviewed (changes requested):
Определиться с тем, что такое лексема, а что — словоформа. Убрать пока что из системных идентификаторов показатели классических частей речи.

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,654 @@
+    <- lexeme_qualitative;lexeme_single;
+    <- lexeme_degree_second_positive;;
+
+l_en_if_do
```
не пишем никаких инфинитивов, а как глагол тогда писать?! Плюс, писал, что не нужно пока писать части речи

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,654 @@
+    (*
+        -> rrel_gramm_aspect_simple: ...
+        (*
+            -> rrel_standard: [TO DO];;
```
А вот в эталоне TO писать уж не стоит

Al-Kom replied:
Эталон должен отражать кусок реального текста сообщения. В main_idtf не стоит, согласен

PlagaMedicum replied:
Да, я там немного запутался. Если словоформа — to do, то и в эталоне пишем TO DO

Al-Kom committed:
Change infinitive lexemes in dialog 4 dictionary

Al-Kom committed:
Update dialog 4 dictionary

Al-Kom commented:
Словарь исправлен. Удалены дублирущиеся лексемы, которые встречаются в других (разобранных) словарях

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,488 @@
+    <- lexeme_taigen; lexeme_minimized;
+    <- lexeme_compressed; lexeme;
+    <- lexeme_informational;lexeme_variable;
+    <- lexeme_arbitrary; <- lexeme_qualitative;
```
codestyle

Al-Kom replied:
What is the code style? Without second '<-' per a line or one node per a line? Please give an example

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,488 @@
+    <- lexeme_taigen; lexeme_minimized;
+    <- lexeme_compressed; lexeme;
+    <- lexeme_informational;lexeme_variable;
+    <- lexeme_arbitrary; <- lexeme_qualitative;
```
fix

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,488 @@
+    <- lexeme_taigen; lexeme_minimized;
+    <- lexeme_compressed; lexeme;
+    <- lexeme_informational;lexeme_constant;
+    <- lexeme_nominative; <-lexeme_qualitative;
```
fix

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,488 @@
+    <- lexeme_taigen; lexeme_minimized;
+    <- lexeme_compressed; lexeme;
+    <- lexeme_informational;lexeme_constant;
+    <- lexeme_nominative; <-lexeme_qualitative;
```
fix

PlagaMedicum commented on kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs:
```diff
@@ -0,0 +1,488 @@
+    <- lexeme_taigen; lexeme_minimized;
+    <- lexeme_compressed; lexeme;
+    <- lexeme_informational;lexeme_constant;
+    <- lexeme_nominative; <-lexeme_qualitative;
```
fix
